### PR TITLE
Remove CLI command `init` now that we have `generate_protocol`

### PR DESCRIPTION
**Pull request by esiegel**

### CODE REVIEW

**Comment by transcripticpll** [comment]
Can you add your change to the changelog? Since we're losing functionality, do we need to do a major version bump?

**Comment by esiegel** [comment]
Can definitely add something to the commit log.  Not sure about bumping to a major version...but could

**Comment by esiegel** [comment]
Rebased the commit onto the release 6.0 branch

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 ## v6.0.0
 Bumping version after adding breaking changes
```
Comment (transcripticpll) [inline CHANGELOG.md]: This comment is extraneous

**Comment by transcripticpll** [comment]
lgtm, thanks for creating the 6.0 branch

**Pull request merged**